submissions/bmukha/dom/app.js

```diff
@@ -0,0 +1,81 @@
+  },
+  {
+    class: 'Warrior',
+    name: 'Garrosh Hellscream',
+    lore: "Garrosh Hellscream, former Warchief of the Horde, is a living testament to the dangers of unchecked aggression Garrosh was born on the orcish homeworld of Draenor, and grew up untainted by the demonic blood that twisted the orcs who invaded Azeroth. Though he wasn't demonically influenced himself, Garrosh was haunted by his family's legacy: his father, Grom Hellscream, was the first orc to embrace corruption by drinking the blood of the pit lord Mannoroth. Garrosh lived in the shadow of his father's acts until he met Warchief Cosmetics - PNG logo.pngThrall, founder of the renewed Horde, who explained to the younger Hellscream how Grom ultimately gave his life to lift the demonic blood-curse, and freed his race from servitude to dark forces Uplifted, Garrosh joined Thrall in Azeroth and quickly became a hero of the Horde by leading a successful offensive against the Lich King. When LegacyDeathwing shattered the world, Thrall was forced to leave his throne to stop the destruction, and left Garrosh as his successor. Looking for ways to gather more resources and new territory for his people, Hellscream has initiated several brutal strikes against the Alliance, razing the city of Theramore and later clashing over the continent of Pandaria. As warchief, Garrosh's controversial actions have exposed the Horde to great scrutiny from outside--and within--his faction. With this increased scrutiny comes a greater danger of dissension and retaliation than the Horde has ever faced before.",
+  },
+];
+
+const renderHero = (hero) => {
+  const heroClass = hero.class.toLowerCase().split(' ').join('');
+  return `<h2 class="hero-name">${hero.name}</h2>
+    <div class="hero-portrait">
+        <img class="hero-portrait_img" src="./img/${heroClass}.webp" alt="heroportrait" />
+    </div>
+    <div class="hero-power">
+        <img class="hero-power_img" src="./img/${heroClass}-heropower.webp" alt="heropower" />
+    </div>
+    <p class="lore">
+        ${hero.lore}
+    </p>`;
+};
+
+const content = document.querySelector('.content');
+
+document.addEventListener('DOMContentLoaded', () => {
+  const randomNumber = Math.floor(Math.random() * 9);
+  content.innerHTML = renderHero(heroes[randomNumber]);
+});
+
+document.querySelector('.menu').addEventListener('click', (event) => {
+  const hero = event.target.parentElement.id;
+  const info = heroes.find(
+    (item) => item.class.toLowerCase().split(' ').join('') === hero
```

Comment:
In your list of heroes one hero with class consists of two words. Maybe you rename and write less code? It only for discussion. How do you think?